autorest/azure/devicetoken.go

```diff
@@ -85,7 +84,7 @@ func InitiateDeviceAuth(client *autorest.Client, oauthConfig OAuthConfig, client
 		autorest.AsPost(),
 		autorest.AsFormURLEncoded(),
 		autorest.WithBaseURL(oauthConfig.DeviceCodeEndpoint.String()),
-		autorest.WithFormData(url.Values{
+		autorest.WithFormData(map[string]interface{}{
 			"client_id": []string{clientID},
```

Comment:
@garimakhulbe I know @colemickens raised this as well, but I'm not sure I see the benefit of moving from `url.Values` to a `map[string]interface{}`. Was there an issue in using `url.Values`.